### ISSUE TITLE
Some questions about the data samples

### ISSUE THREAD

**Issue by HBlueNet** [YulongBonjour/STTM#4]
Thank you for your excellent work. I have a small question regarding the NSD dataset. According to the NSD documentation, there are a total of 73k visual images, among which 1k are shared. That means each subject used a total of 10k natural images, consisting of 9k individual images plus 1k shared ones. However, in your paper, it is described that each subject's training set contains 8,859 natural images and the test set contains 982 natural images. Could you please explain if any filtering was performed or if there are differences in the dataset used?Thanks

**Comment by YulongBonjour** [comment]
Thanks for your question. At the beginning, The NSD dataset only released 38 session. Recently, they released all 40 sessions. That is the reason.  Prior works like Brain-diffuser used the old version.Recent work MindEye2 updated their performance with the full dataset.

**Comment by HBlueNet** [comment]
Thank you for your reply. For the purpose of reproducing results and facilitating comparisons—since I noticed that the backbone of the CLIP model used in this paper differs from that in BrainCLIP, and I aim to reproduce both papers simultaneously—could you please provide the visual images and captions data from the old version? Alternatively, do you have the nsd_stim_info_merged.csv file corresponding to the visual images of the old version? This way, I can identify the old-version visual images by myself and find the corresponding captions from the MSCOCO dataset.

**Comment by YulongBonjour** [comment]
I will check and reply you later.  I am a little busy now.

**Comment by HBlueNet** [comment]
No problem, and thank you for your help.

**Comment by YulongBonjour** [comment]
I remember I used this script to download the 38 sessions fMRI. https://github.com/ozcelikfu/brain-diffuser/blob/main/data/download_nsddata.py    . You can change the number to 40 to download all the sessions.

**Comment by HBlueNet** [comment]
Help a lot. I will try to collect the data of old version. Thanks for your Help again.I will close this issue.
